docs/app/pages/components/components-sections/media-player/media-player/media-player.component.ts

```diff
@@ -26,7 +27,7 @@ export class ComponentsMediaPlayerComponent extends BaseDocumentationSection imp
             'app.component.css': this.snippets.raw.appCss
         },
         modules: [{
-            imports: ['RadioButtonModule', 'MediaPlayerModule', 'AccordionModule'],
+            imports: ['RadioButtonModule', 'MediaPlayerModule', 'AccordionModule', 'SelectModule', 'FormsModule', 'ReactiveFormsModule'],
```

Comment:
Are these additional imports required? None of the changes to the template use a select component or reactive forms?